script/release/get-url-hash.js

```diff
@@ -8,24 +8,32 @@ const lambda = new AWS.Lambda({
   region: 'us-east-1'
 });
 
-module.exports = function getUrlHash (targetUrl, algorithm = 'sha256') {
-  return new Promise((resolve, reject) => {
-    lambda.invoke({
-      FunctionName: 'hasher',
-      Payload: JSON.stringify({
-        targetUrl,
-        algorithm
-      })
-    }, (err, data) => {
-      if (err) return reject(err);
-      try {
-        const response = JSON.parse(data.Payload);
-        if (response.statusCode !== 200) return reject(new Error('non-200 status code received from hasher function'));
-        if (!response.hash) return reject(new Error('Successful lambda call but failed to get valid hash'));
-        resolve(response.hash);
-      } catch (err) {
-        return reject(err);
-      }
+module.exports = async function getUrlHash (targetUrl, algorithm = 'sha256', attempts = 3) {
```

Comment:
There is no need marking this function as `async` since it is returning a Promise.